test/mciproject.yml

```diff
@@ -184,11 +834,56 @@ buildvariants:
   - ubuntu1404-test
   expansions:
     # It's ugly, but we need the absolute path here, not the relative
-    test_env_vars: LD_LIBRARY_PATH=`pwd`/.libs
+    test_env_vars: PATH=/opt/mongodbtoolchain/v2/bin:$PATH LD_LIBRARY_PATH=$(pwd)/LOCAL_INSTALL/lib
+    test_env_vars_py: PATH=/opt/mongodbtoolchain/v2/bin:$PATH LD_LIBRARY_PATH=$(pwd)/../lib
     smp_command: -j $(grep -c ^processor /proc/cpuinfo)
-    configure_env_vars: CC=/opt/mongodbtoolchain/bin/gcc CXX=/opt/mongodbtoolchain/bin/g++
+    configure_env_vars: CC=/opt/mongodbtoolchain/bin/gcc CXX=/opt/mongodbtoolchain/bin/g++ PATH=/opt/mongodbtoolchain/v2/bin:$PATH
   tasks:
     - name: compile
+    - name: lang-python-test
+    - name: examples-c-test
+    - name: bloom-test
+    - name: checkpoint-test
+    - name: cursor-order-test
+    - name: fops-test
```

Comment:
Does the scope of this new `fops-test` task cover that of the old `fops` task? The new task ran much longer time, and the old task has more verbose log output.